### PR TITLE
Adds impersonate privilege to the glossary and 4.4 additions

### PR DESCRIPTION
**Pull request by ncordon**
Corrects broken links.

Adds `IMPERSONATE` privilege to the glossary of terms and the new additions to 4.4

### CODE REVIEW

File: cypher/cypher-docs/src/docs/dev/glossary.asciidoc [hunk 1]
```diff
@@ -374,27 +374,28 @@ The following commands are only executable against the `system` database:
...
 | <<administration-security-reads-traverse, TRAVERSE>>                                                    | GRAPH                 | Determines whether an element (node, relationship or both) can be found and traversed while executing queries on the specified graph.
-| <<administration-security-administration-dbms-privileges-user-management, USER MANAGEMENT>>             | DBMS                  | Determines whether the user can create, drop, modify and show users.
+| <<access-control-dbms-administration-user-management, USER MANAGEMENT>>             | DBMS                  | Determines whether the user can create, drop, modify and show users.
 | <<administration-security-writes-write, WRITE>>                                                         | GRAPH                 | Determines whether the user can execute write operations on the specified graph.
+| <<access-control-dbms-administration-impersonation, IMPERSONATE>>                   | DBMS              | Determines whether a user can impersonate another one and assume their privileges.
```
Comment (mnd999) [inline cypher/cypher-docs/src/docs/dev/glossary.asciidoc]: I think these are supposed to be in alphabetical order, so this one seems out of place.

File: cypher/cypher-docs/src/docs/dev/glossary.asciidoc [hunk 2]
```diff
@@ -374,27 +374,28 @@ The following commands are only executable against the `system` database:
...
-| <<administration-security-administration-dbms-privileges-role-management, REMOVE ROLE>>                 | DBMS                  | Determines whether the user can revoke roles.
-|<<administration-security-administration-dbms-privileges-role-management, RENAME ROLE>>                  | DBMS                  | Determines whether the user can rename roles.
-|<<administration-security-administration-dbms-privileges-user-management, RENAME USER>>                  | DBMS                  | Determines whether the user can rename users.
-| <<administration-security-administration-dbms-privileges-role-management, ROLE MANAGEMENT>>             | DBMS                  | Determines whether the user can create, drop, grant, revoke and show roles.
+| <<access-control-dbms-administration-privilege-management, REMOVE PRIVILEGE>>       | DBMS                  | Determines whether the user can remove privileges using the REVOKE command.
+| <<access-control-dbms-administration-role-management REMOVE ROLE>>                 | DBMS                  | Determines whether the user can revoke roles.
```
Comment (mnd999) [inline cypher/cypher-docs/src/docs/dev/glossary.asciidoc]: I think you are missing a comma.

**Review by mnd999 — CHANGES_REQUESTED** [review]

**Review by mnd999 — APPROVED** [review]
Looks good now

**Review by davidoliverSP2 — APPROVED** [review]
Thank @ncordon 

👍

**Pull request merged**